src/server/users/router.js

```diff
@@ -121,7 +121,14 @@ router.get('/createdposts', requiredPermission(PERMISSION.R), controller.getCrea
  *       error: "database error"
  *     }
  */
-router.get('/:id', requiredPermission(PERMISSION.R), controller.getById);
+router.get('/:id', () => {
```

Comment:
What is headerSent?
if controller send something like 
`res.send({message: 'success'});`
res got this 'headerSent' field?